bundles/pom.xml

```
@@ -200,6 +200,7 @@
     <module>org.openhab.binding.luftdateninfo</module>
     <module>org.openhab.binding.lutron</module>
     <module>org.openhab.binding.luxtronikheatpump</module>
+    <module>org.openhab.binding.luxom</module>
```

This will keep the modules sorted:

```suggestion
    <module>org.openhab.binding.luxom</module>
    <module>org.openhab.binding.luxtronikheatpump</module>
```